### DISCUSSION TITLE
GNSS orbit prediction

### DISCUSSION THREAD

**Discussion by Yang-Hu-LNTU** [groops-devs/groops#80] [Q&A] [answered]
### Description

How to forecast a few hours before and after the daily precise orbit products？Or how to modify the source code to achieve the above purpose

### Operating systems

- [X] Linux
- [ ] Windows
- [ ] macOS
- [ ] Other (please specify in the description)

**Comment by akvas** [comment]
Dear @Yang-Hu-LNTU 

we decided to set up GitHub discussions for usage and processing related questions. The motivation behind this step was that discussions remain visible, while issues are hard to keep track of once they are closed. I encourage you to continue this thread, or post your next question in the [Q&A section](https://github.com/groops-devs/groops/discussions/categories/q-a) of our discussion board. Thank you!

Cheers,
Andreas

**Comment by tmayerguerr** [comment, marked as answer]
Maybe you can use SimulateOrbit with the integration constants from your estimated orbit file. To the standard forces you may add miscAccelerations->fromParametrization to add the estimated ECOM parameters.
regards
Torsten
